curp-test-utils/src/lib.rs

```diff
@@ -1,8 +1,5 @@
 #![allow(dead_code, unused)]
```

Comment:
I think we didn't need this, please remove it and remove dead codes

Reply:
Done